README.md

```diff
@@ -55,7 +60,6 @@ b'\x01\x88?\x00'
 >>> mock_panel.stop()
 >>> mockthread.kill()
 >>> mockthread.join()
-
```

Comment:
This empty line is actually required for the doctest to pass

```
___________________________________________________________________ [doctest] README.md ____________________________________________________________________
053 >>> panel_control.connect()
054 >>> mockthread.start()
055 >>> panel_control.set_text("test01")
056 >>> panel_control.get_text()
057 'test01     '
058 >>> panel_control.get_serial_number(10)
059 b'\x01\x88?\x00'
060 >>> mock_panel.stop()
061 >>> mockthread.kill()
062 >>> mockthread.join()
Expected:
    ```
Got nothing

/home/fuj/repos/github.com/adfinis/sbb-fallblatt/README.md:62: DocTestFailure
```

Reply:
Fixed, thanks!